src/CodeExplorerBundle/Resources/config/services.yml

```diff
@@ -1,9 +1,9 @@
 services:
+    _defaults: { public: false }
+
     code_explorer.twig.source_code_extension:
-        public:    false
-        class:     CodeExplorerBundle\Twig\SourceCodeExtension
-        tags:
-            - { name: twig.extension }
+        class: CodeExplorerBundle\Twig\SourceCodeExtension
+        tags: [twig.extension]
```

Comment:
Well, AppBundle was migrated to use all new features. I think CodeExplorerBundle should be updated too